.evergreen/run-tests.sh

```diff
@@ -53,7 +53,7 @@ if [[ -z "${CLIENT_ENCRYPTION}" ]]; then
   unset AWS_ACCESS_KEY_ID;
   unset AWS_SECRET_ACCESS_KEY;
 else
-  npm install mongodb-client-encryption
+  npm install mongodb-client-encryption@1.1.1-beta.0
```

Comment:
Is this to ensure we get in the (semi) recent changes to add GCP and Azure? I think we'd be better off using a range so we can get the latest as we make more releases: `npm install mongodb-client-encryption@">=1.2.1"`

Reply:
That change came in from the 3.6 PR (#2662) so I'm not entirely sure, but agreed using a range is better.

Reply:
Ah my mistake, I missed that came from there we should correct it there too.